packages/eslint-plugin/src/utils/runtime-lint-utils.ts

```diff
@@ -533,6 +564,31 @@ function getAngularEntryPoint(file: string, projectRoot: string): string {
   return undefined;
 }
 
+function getPackageEntryPoints(
+  projectRoot: string
+): Array<{ path: string; file: string }> {
+  const packageContent = readFileIfExisting(
+    path.join(workspaceRoot, projectRoot, 'package.json')
+  );
+  if (!packageContent) {
+    return [];
+  }
+  const exports = parseJson(packageContent).exports;
+  if (!exports) {
+    return [];
+  }
+  const entryPaths: Array<{ path: string; file: string }> = [];
+  for (const [key, value] of Object.entries<string>(exports)) {
```

Comment:
The `exports` property [could be a `string`](https://nodejs.org/docs/latest-v22.x/api/packages.html#exports-sugar).

Reply:
Thanks, let me rework that.